spec/presenters/finder_presenter_spec.rb

```diff
@@ -248,6 +248,17 @@ def sort_option(label, value, disabled: false, selected: false)
     end
   end
 
+  describe "#all_content_finder?" do
+    fit 'returns true if finder is "All Content"' do
```

Comment:
What does `fit` do?

Reply:
Its a Fit bit (https://www.fitbit.com/) but only for specs :trollface: 

it only used to run specific 'focused' tests.

Good spot.